### DISCUSSION TITLE
大佬，帮忙看下这个跨域的错误，该如何修复的？

### DISCUSSION THREAD

**Discussion by hackhase** [lisonge/vite-plugin-monkey#134] [Q&A]
Refused to load the script 'http://127.0.0.1:5173/__vite-plugin-monkey.entry.js' because it violates the following Content Security Policy directive: "script-src 'self' blob: *.citysbs.com *.19lou.com *.cqmmgo.com *.19louimg.cn *.baidu.com *.baidustatic.com api.map.baidu.com *.bdstatic.com *.pstatp.com c.mipcdn.com tjs.sjs.sinajs.cn c.cnzz.com  s22.cnzz.com res.wx.qq.com apis.map.qq.com c.dun.163.com cstaticdun.126.net s11.cnzz.com static.geetest.com api.geetest.com *.alicdn.com  *.bdimg.com c.dun.163yun.com jsapi.qq.com mat1.gtimg.com analytics.snssdk.com app.citybrain.hangzhou.gov.cn 'unsafe-inline' 'unsafe-eval'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

**Comment by 1uan3hu** [comment]
这个问题是浏览器csp机制导致的，可以用up主的另外一个开源项目解决

https://github.com/lisonge/Disable-CSP
